react-app/src/native/components/leagues/TeamList.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Text, Card, CardItem} from 'native-base';
+import { Table, Row } from 'react-native-table-component';
+import { StyleSheet } from 'react-native';
+import Loading from '../common/Loading';
+
+export default class Schedule extends React.Component {
+
```

Comment:
I noticed this in Schedule.js as well, you're missing propTypes and defaultProps from your class specifications. They're helpful for letting coders know what props are available to them inside the class. Also you can specify a prop as required and if the value isn't passed through an error is thrown so you know at render time you'll have all the props you need.